qiskit/primitives/statevector_estimator.py

```diff
@@ -35,6 +35,61 @@ class StatevectorEstimator(BaseEstimatorV2):
     pure state vectors. These states are subsequently acted on by :class:~.SparsePauliOp`,
     which implies that, at present, this implementation is only compatible with Pauli-based
     observables.
+
+    Each tuple of ``(circuit, observables, <optional> parameter values, <optional> precision)``,
+    called an estimator primitive unified bloc (PUB), produces its own array-based result. The
+    :meth:`~.EstimatorV2.run` method can be given many pubs at once.
+
+    .. code: python
```

Comment:
If you want to render the plot you can do:

```suggestion
    .. plot::
         :include-source:
```

You might have to break out the last line into a standalone code block though.

Reply:
Thanks for the tip.